e2e/test/scenarios/question/summarization.cy.spec.js

```diff
@@ -71,24 +71,24 @@ describe("scenarios > question > summarize sidebar", () => {
     });
   });
 
-  it("selected dimensions from another table includes the table name when becomes pinned to the top", () => {
+  it("selected dimensions from another table includes the table alias when becomes pinned to the top", () => {
     getDimensionByName({ name: "State" }).click();
 
     cy.button("Done").click();
 
     summarize();
 
     cy.findByTestId("pinned-dimensions").within(() => {
-      getDimensionByName({ name: "People → State" }).should(
+      getDimensionByName({ name: "User → State" }).should(
```

Comment:
This is a valid change:
<img width="237" alt="Screen Shot 2023-03-16 at 11 21 21 AM" src="https://user-images.githubusercontent.com/14301985/225647251-68cabd06-1d67-438f-8be7-b5833ae34f6c.png">